webpack/assets/javascripts/react_app/components/ConfigReports/ConfigReports.js

```diff
@@ -38,15 +40,15 @@ const ConfigReports = props => {
         />
       </Col>
 
-      <Col md={4}>
+      <Col md={5} className="bar-chart-medium-width">
         <ChartBox
           {...chartBoxProps}
           type="bar"
           chart={{ data: statusChartData, id: 'report-status' }}
           title={__('Report Status')}
         />
       </Col>
-      <Col md={4}>
+      <Col md={2}>
```

Comment:
This makes it very hard to read on a smaller screen:
![Screen Shot 2020-06-01 at 15 53 36](https://user-images.githubusercontent.com/155810/83415871-1ebf8c80-a420-11ea-94c3-a576b7171f47.png)